oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsAggregateWorker.java

```diff
@@ -69,7 +69,7 @@
                                                           .getService(MetricsCreator.class);
         aggregationCounter = metricsCreator.createCounter(
             "metrics_aggregation", "The number of rows in aggregation",
-            new MetricsTag.Keys("metricName", "level", "dimensionality"), new MetricsTag.Values(modelName, "1", "min")
+            new MetricsTag.Keys("metricName", "level", "dimensionality"), new MetricsTag.Values(modelName, "1", "minute")
```

Comment:
Why Change this?

Reply:
Keep code style uniform.

Reply:
@hanahmily You made the decision. I am not sure whether this is a suitable change. Basically, we don't have code style guidance about this.

Reply:
From my perspective, some other references should be updated accordingly. One I can find is `grafana.json` at docs folder. But it's a bit overkill. I prefer to leave it alone

Reply:
@Ax1an Pleasw update the Grafana config to adopt this.

Reply:
From my perspective, minute is better than min. Because DownSampling enum is minute.
I prefer to adjust grafana.json.
![image](https://user-images.githubusercontent.com/28091237/86510869-6160f400-be26-11ea-9e17-38b8d8e99d0f.png)

Reply:
@wu-sheng Updated.

Reply:
After a talk with sheng, I agree with renaming it. Pls, to search and update other possible references more than `grafana.json`. Thanks.

Reply:
@hanahmily I updated oap.yaml. Other references were not found.

Reply:
These are the metrics in grafana after this PR changed.
![image](https://user-images.githubusercontent.com/28091237/86554452-3a1e3a00-bf80-11ea-9745-3e0ba12dce76.png)
![image](https://user-images.githubusercontent.com/28091237/86554455-3d192a80-bf80-11ea-850c-77b095396eb7.png)